packages/assessment/src/pages/PastExaminations.js

```diff
@@ -0,0 +1,159 @@
+} from "native-base";
+import colorTheme from "../colorTheme";
+const colors = overrideColorTheme(colorTheme);
+
+export default function PastExaminationsList({
+  classId,
+  selectedSubject,
+  schoolDetails,
+}) {
+  const { t } = useTranslation();
+  const navigate = useNavigate();
+  const [width, height] = useWindowSize();
+  const [loading, setLoading] = useState(false);
+  const [progressAssessment, setProgressAssessment] = React.useState([
+    {
+      name: "12 Assessed",
+      color: colors.successBarColor,
+      value: 12,
+    },
+    {
+      name: "6 pending",
+      color: colors.pendingBarColor,
+      value: 6,
+    },
+  ]);
+
+  if (loading) {
+    return <Loading height={height - height / 2} />;
+  }
+
+  return (
+    <Layout
+      _header={{
+        title: "Past Examinations",
+      }}
+      _appBar={{
+        languages: ["en"],
+        // onPressBackButton: handleBackButton,
+      }}
+      subHeader={
+        <VStack>
+          <H2>{selectedSubject}</H2>
+          <HStack alignItems={"center"}>
+            <Caption color={colors.gray}>
+              {schoolDetails && schoolDetails.name}
+            </Caption>
+            {schoolDetails && schoolDetails.section && (
+              <Caption color={colors.gray}> {schoolDetails.section}</Caption>
+            )}
+          </HStack>
+        </VStack>
+      }
+      _subHeader={{ bg: colors.cardBg, py: "6" }}
+      _footer={{
+        menues: [
+          {
+            title: "HOME",
+            icon: "Home4LineIcon",
+            module: "Registry",
+            route: "/",
+            routeparameters: {},
+          },
+          {
+            title: "CLASSES",
+            icon: "TeamLineIcon",
+            module: "Registry",
+            route: "/classes",
+            routeparameters: {},
+          },
+          {
+            title: "SCHOOL",
+            icon: "GovernmentLineIcon",
+            module: "Registry",
+            route: "/",
+            routeparameters: {},
+          },
+          {
+            title: "MATERIALS",
+            icon: "BookOpenLineIcon",
+            module: "Registry",
+            route: "/",
+            routeparameters: {},
+          },
+          {
+            title: "CAREER",
+            icon: "UserLineIcon",
+            module: "Registry",
+            route: "/",
+            routeparameters: {},
+          },
+        ],
+      }}
+    >
+      <Box p={4}>
+        <>
+          <VStack space={6}>
+            <Box
+              borderWidth="1"
+              borderColor={colors.borderColor}
+              borderRadius="10px"
+            >
+              <VStack space="4">
+                <Box p="4" pb="4px" roundedTop="6">
+                  <VStack space={4}>
+                    <Box>
+                      <BodyLarge py="2">
+                        {t("Summative Assessment 1")}
+                      </BodyLarge>
+                      <BodySmall color={colors.gray}>27, May 2022</BodySmall>
```

Comment:
This date shouldn't be hard coded

Reply:
This is static page as it is part of state examination, Sagar will take this up while API binding of state examination flow.